test/command/2662.md

```diff
@@ -7,5 +7,5 @@
    :scale: 300 %
    :alt: alternate text
 ^D
-<p><img src="http://url.to.image/foo.png" alt="alternate text" class="align-left" width="600" height="300" /></p>
```

Comment:
Some change seems to have switched the position of the alt attribute relative to the others.  I think it would be desirable not to introduce this change, if possible, since it has no essential connection to the issue being solved here and might conceivably create problems for others (e.g. in test suites).

Reply:
Yes, it would be possible by making the haskell source code more complicated and I didn't think it was worth the trade-off. But I could change it of course...